openmdao/core/indepvarcomp.py

```diff
@@ -39,6 +39,16 @@ def __init__(self, name=None, val=1.0, **kwargs):
         if isinstance(name, str):
             super().add_output(name, val, **kwargs)
 
+        elif isinstance(name, (list, tuple)):
+            for tup in name:
+                if len(tup) == 2:
```

Comment:
You might want to verify here that tup is actually a tuple, or at least not a string. Otherwise in cases where the user gives you a list/tuple of strings, you'll get weird error messages.  I could see a user just providing a list of strings and assuming those variables would all have default values.

Reply:
Good call, I've modified this and pushed up!

Reply:
I think the docstring for `name` needs to be updated for these new possibilities...

Also maybe some kind of error checking on the `tup` values (i.e. tup[0] must be a string, tup[2] must be a dict?)?

Reply:
Thanks Steve, I've updated the docstring and added type checking!